### PR TITLE
Remove unused gRPC methods from ingest servicer

### PR DESCRIPTION
**Pull request by jotare**
### Description
Describe the proposed changes made in this PR.

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

**Review by hermeGarcia — COMMENTED** [review]
Rust part is good! 
Less code == ❤️

File: nucliadb/nucliadb/common/cluster/base.py [hunk 1]
```diff
@@ -109,11 +109,6 @@ async def delete_shard(self, id: str) -> str:
         resp: noderesources_pb2.ShardId = await self.writer.DeleteShard(req)  # type: ignore
         return resp.id
 
-    async def clean_and_upgrade_shard(self, id: str) -> noderesources_pb2.ShardCleaned:
-        req = noderesources_pb2.ShardId(id=id)
-        resp = await self.writer.CleanAndUpgradeShard(req)  # type: ignore
-        return resp
-
     async def del_vectorset(self, shard_id: str, vectorset: str) -> OpStatus:
```
Comment (lferran) [inline nucliadb/nucliadb/common/cluster/base.py:112]: Should these ones be deleted too?

File: nucliadb/nucliadb/ingest/tests/integration/servicer/test_knowledgebox_servicer.py [hunk 2]
```diff
@@ -56,33 +53,32 @@ async def test_create_knowledgebox(grpc_servicer: IngestFixture, maindb_driver):
     result = await stub.NewKnowledgeBox(pb)  # type: ignore
     assert result.status == knowledgebox_pb2.KnowledgeBoxResponseStatus.OK
 
-    pb_prefix = knowledgebox_pb2.KnowledgeBoxPrefix(prefix="test")
-    slugs = []
-    async for kpb in stub.ListKnowledgeBox(pb_prefix):  # type: ignore
-        slugs.append(kpb.slug)
-
+    slugs = await list_kb_shards(maindb_driver)
```
Comment (lferran) [inline nucliadb/nucliadb/ingest/tests/integration/servicer/test_knowledgebox_servicer.py:56]:
```suggestion
    slugs = await list_kb_slugs(maindb_driver)
```

**Review by lferran — APPROVED** [review]

File: nucliadb/nucliadb/common/cluster/base.py (hunk 1 above)
Comment (jotare) [reply nucliadb/nucliadb/common/cluster/base.py:112]: I was going to split it in another PR to not mix cleans, but I can do it here too

**Comment by jotare** [comment]
[sc-8991]

**Pull request merged**